### PR TITLE
Eslint fixes in backend

### PR DESCRIPTION
**Pull request by rmoestl**

### CODE REVIEW

File: backend/main.development.js [hunk 1]
```diff
@@ -43,6 +42,7 @@ app.on('ready', () => {
     httpsServer.electronWindow = mainWindow;
 
     const HTTP_URL = `${httpServer.protocol}://${httpServer.domain}:${httpServer.port}`;
+    // TODO ESLint no-unused-vars: though not used, would it be better to use HTTPS?
     const HTTPS_URL = `${httpsServer.protocol}://${httpsServer.domain}:${httpsServer.port}`;
```
Comment (chriddyp) [inline backend/main.development.js]: hm, if this isn't used then we can remove it. HTTPS happens in a variety of different ways

File: backend/persistent/PlotlyAPI.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export function newDatacache(payloadJSON, type) {
     }).then(res => {
         return res;
     }).catch(err => {
-        console.error(err);
+        console.error(err); // eslint-disable-line no-console
```
Comment (chriddyp) [inline backend/persistent/PlotlyAPI.js]: Let's actually replace this with a `Logger.log` call (same goes for the rest of the backend error handling)

**Comment by chriddyp** [comment]
💃

File: backend/main.development.js (hunk 1 above)
Comment (rmoestl) [reply backend/main.development.js]: Thx, will remove it.

File: backend/persistent/PlotlyAPI.js (hunk 2 above)
Comment (rmoestl) [reply backend/persistent/PlotlyAPI.js]: Alright, I'll replace 'em.

**Pull request merged**